module/platform/darwin/HardwareIO/tests/ConversionTest.cpp

```diff
@@ -146,7 +147,7 @@ TEST_CASE("Testing the hardware position conversions to radians", "[hardware][co
                                                            {3073, M_PI_2},
                                                            {4095, M_PI}};
 
-        for (size_t i = 0; i < 20; ++i) {
+        for (uint8_t i = 0; i < 20; ++i) {
```

Comment:
Why uint8_t? just make it an int.
uint8_t doesn't make anything faster, it still has to go in a register anyway

Reply:
It's not to make it faster, it's because `Convert::servoPosition` takes a `uint8_t`. The variable was throwing a narrowing conversion warning IIRC, and since it's a fixed size loop with fewer than 256 iterations, I figured the cleanest solution was just to make the loop variable the same type. The other option would be to explicitly cast it to that type when that function is called, and this just seemed nicer

Reply:
unsigned arithmetic is actually slower as the compiler has to generate extra code to check for overflows (not sure if it turns that off if it knows the bounds or not). But I suppose that given this is a test we dont really care about this performance hit here

Reply:
Interesting point that it's slower generally. I would be interested in reading more about that.
In this case though, I am quite confident that since there is a fixed, small number of iterations, it would unroll the loop anyway and do away with the loop variable entirely